backend/src/test/java/com/example/backend/service/EquipmentItemServiceTest.java

```diff
@@ -145,6 +146,76 @@ void updateEquipmentItem() {
 
         //then
         EquipmentItem expected = testItem;
+        verify(equipmentItemRepository).save(testItem);
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    void deleteEquipmentItem_whenValid_thenReturnItemID() {
+        //given
+        EquipmentItem testItem = EquipmentItem.builder()
+                .id("TestID")
+                .title("testtitle")
+                .description("testdescription")
+                .owner("testownerID")
+                .done(false)
+                .important(false)
+                .build();
```

Comment:
Is it for this test necessary to use all these properties? Could it be simpler
```suggestion
                .id("TestID")
                .build();
```

You know, less to read in the future...However, I did not test to get sure the properties are needed for this test.

Reply:
In deed, they are not 😄 . But as you see, I reuse the testdata and just copy them in the new test. I personally like to test with more "realistic" data, but it's a personal thing 😉